### DISCUSSION TITLE
Whitelist for pipeline approvals

### DISCUSSION THREAD

**Discussion by pat-s** [woodpecker-ci/woodpecker#2489] [Request a Feature] [answered]
It would be awesome if there would be an option to whitelist some users for pipeline approvals, i.e. PRs opened by these do not wait for manual approval.

This would be great to have for bots like `renovate` but also for "trusted" parties, e.g. maintainers of a project.

**Comment by qwerty287** [comment, marked as answer]
That's a duplicate of https://github.com/woodpecker-ci/woodpecker/issues/336 (and also related: https://github.com/woodpecker-ci/woodpecker/discussions/2293)
